ru_RU.lang

```diff
@@ -209,35 +209,35 @@ replaymod.gui.rendersettings.addtoqueue=Добавить в очередь
 replaymod.gui.rendersettings.renderer.cubic=Кубический
-replaymod.gui.rendersettings.renderer.equirectangular=Равнопромежуточная
+replaymod.gui.rendersettings.renderer.equirectangular=Равноугольный
 replaymod.gui.rendersettings.renderer.ods=ODS
-replaymod.gui.rendersettings.renderer.blend=Экспорт в Blender [Экспериментально]
+replaymod.gui.rendersettings.renderer.blend=Экспорт в Blender [БЕТА]
 
 replaymod.gui.rendersettings.renderer.default.description=Видео в указанном разрешении. Самый быстрый вариант.
 replaymod.gui.rendersettings.renderer.stereoscopic.description=Стереоскопическое 3D-видео (метод перекрёстного взгляда). Изображение для одного глаза составляет половину ширины видео.
 replaymod.gui.rendersettings.renderer.cubic.description=Видео с 360-градусным панорамным видом, использующее кубическую карту. Его можно воспроизвести в Oculus Rift, а также в некоторых видеоплеерах (например, в VR Player).
 replaymod.gui.rendersettings.renderer.equirectangular.description=Видео с 360-градусным панорамным видом, использующее равнопромежуточную проекцию. Его можно воспроизвести в YouTube, Oculus Rift, а также в некоторых видеоплеерах (например, в VR Player).
 replaymod.gui.rendersettings.renderer.ods.description=Видео со стереоскопическим 360-градусным панорамным видом, также известное как VR-видео. Его можно воспроизвести в YouTube.
-replaymod.gui.rendersettings.renderer.blend.description=Видимые контуры, экспортированные в .blend-файл, который можно открыть в Blender. Крайне нестабильная функция! Требуется большой объём памяти, содержит довольно много ошибок, иногда генерирует повреждённые файлы.
+replaymod.gui.rendersettings.renderer.blend.description=Экспортирует видимую геометрию как файл .blend, который может быть открыт используя Blender. Экспериментальная функция: Требует много ОЗУ, есть ошибки и иногда на выходе образуются нерабочие файлы.
 
-replaymod.gui.rendersettings.customresolution=Разрешение видео
+replaymod.gui.rendersettings.customresolution=Разрешение Видео
 replaymod.gui.rendersettings.customresolution.warning.yuv420=Длина и ширина должны быть чётными числами.
 replaymod.gui.rendersettings.customresolution.warning.yuv420.cubic=Высота должна быть кратна 6
-replaymod.gui.rendersettings.customresolution.warning.cubic.height=При кубическом рендеринге высота должна быть кратна 3
+replaymod.gui.rendersettings.customresolution.warning.cubic.height=Высота должна быть кратна 3
 replaymod.gui.rendersettings.bitrate=Битрейт
 replaymod.gui.rendersettings.framerate=Частота кадров
 replaymod.gui.rendersettings.chromakey=Хромакей
 replaymod.gui.rendersettings.nametags=Отображать имена
-replaymod.gui.rendersettings.includealpha=Сохранить альфа-канал
+replaymod.gui.rendersettings.includealpha=Сохранить Альфа-канал
 replaymod.gui.rendersettings.outputfile=Выходной файл
-replaymod.gui.rendersettings.sphericalFov=Сферическое поле зрения
+replaymod.gui.rendersettings.sphericalFov=Сферический FOV
```

Comment:
The game itself seems to be using the original value for this one:
![](https://i.johni0702.de/a9jT9.png)
Is there a good reason you changed it or should we stick to what Minecraft does?

Reply:
I changed it because it probably doesn't fit in replaymod's ui

Reply:
It just about fits, so I'm gonna keep it:
![](https://i.johni0702.de/eabNw.png)